dozer-integrations/dozer-spring-support/dozer-spring4/src/test/resources/springNameSpace.xml

```diff
@@ -21,7 +21,7 @@
        xmlns:dozer="http://dozermapper.github.io/schema/dozer-spring"
        xsi:schemaLocation="
           http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-4.3.xsd
-          http://dozermapper.github.io/schema/dozer-spring http://dozermapper.github.io/schema/dozer-spring.xsd">
+          http://dozermapper.github.io/schema/dozer-spring https://dozermapper.github.io/schema/dozer-spring.xsd">
```

Comment:
Can you add a test for this. i know HTTPS will work, but i dont want to suggest something thats not working. see:
- https://github.com/DozerMapper/dozer/pull/538#issuecomment-346020929

Reply:
Added test.